.config/tsaoptions.json

```
@@ -1,10 +1,10 @@
-{  
+{
     "instanceUrl": "https://devdiv.visualstudio.com/",
     "template": "TFSDEVDIV",
     "projectName": "DEVDIV",
     "areaPath": "DevDiv\\NET Fundamentals\\Infrastructure\\Arcade\\SDL",
     "iterationPath": "DevDiv",
     "notificationAliases": [ "dnceng@microsoft.com" ],
-    "repositoryName":"xliff-tasks",
-    "codebaseName": "xliff-tasks"
+    "repositoryName":"dotnet-scenario-tests",
+    "codebaseName": "dotnet-scenario-tests"
```

I think this should just be "scenario-tests". See https://github.com/dotnet/runtime/blob/05fa8813a7774890b46e60109d8cd6d4c853a014/.config/tsaoptions.json#L8-L9

---

I found examples of both cases but why not
```suggestion
    "repositoryName":"scenario-tests",
    "codebaseName": "scenario-tests"
```